src/client/sandbox/native-methods.js

```diff
@@ -39,10 +39,13 @@ class NativeMethods {
         this.documentCreateTouchList     = doc.createTouchList || docProto.createTouchList;
 
         // getters/setters
-        const docPrototype = win.Document.prototype;
+        const docPrototype         = win.Document.prototype;
+        const htmlDocPrototype     = win.HTMLDocument.prototype;
+        const documentAllPropOwner = docPrototype.hasOwnProperty('all') ? docPrototype : htmlDocPrototype;
```

Comment:
Move to `utils/feature-detection.js` as `featureDetection.getDocumentAllPropertyOwner` function

Reply:
We cannot import the `feature-detection` module into the `nativeMethods` because it is circular dependency.

Reply:
Sorry. ‘NativeMethods.getDocumentAllPropertyOwner’